lib/wasmer-types/src/entity/packed_option.rs

```diff
@@ -0,0 +1,171 @@
+//! Compact representation of `Option<T>` for types with a reserved value.
+//!
+//! Small Cranelift types like the 32-bit entity references are often used in tables and linked
```

Comment:
Do we want to generalize the doc comments? We use these types for more than just interacting with Cranelift.

Reply:
:+1: